vault/acl.go

```diff
@@ -6,22 +6,18 @@ import (
 )
 
 var (
-	// Policy lists are used to restrict what is eligible for an operation
-	anyPolicy     = []string{PathPolicyDeny}
-	readWriteSudo = []string{PathPolicyRead, PathPolicyWrite, PathPolicySudo}
-	writeSudo     = []string{PathPolicyWrite, PathPolicySudo}
-
 	// permittedPolicyLevel is used to map each logical operation
-	// into the set of policies that allow the operation.
+	// into the set of capabilities that allow the operation.
 	permittedPolicyLevels = map[logical.Operation][]string{
-		logical.ReadOperation:     readWriteSudo,
-		logical.UpdateOperation:    writeSudo,
-		logical.DeleteOperation:   writeSudo,
-		logical.ListOperation:     readWriteSudo,
-		logical.HelpOperation:     anyPolicy,
-		logical.RevokeOperation:   writeSudo,
-		logical.RenewOperation:    writeSudo,
-		logical.RollbackOperation: writeSudo,
+		logical.CreateOperation:   []string{CreateCapability},
+		logical.ReadOperation:     []string{ReadCapability},
+		logical.UpdateOperation:   []string{UpdateCapability},
+		logical.DeleteOperation:   []string{DeleteCapability},
+		logical.ListOperation:     []string{ListCapability},
+		logical.HelpOperation:     []string{CreateCapability, ReadCapability, UpdateCapability, DeleteCapability, ListCapability, SudoCapability, DenyCapability},
```

Comment:
Given that we special case Help anyways, I wonder if we should just make this the empty list and put a comment? I'm fine either way